hack/istio/tempo/install-tempo-env.sh

```diff
@@ -9,7 +9,7 @@
 #
 ##############################################################################
 
-CLIENT_EXE="kubectl"
+CLIENT_EXE="oc"
```

Comment:
```suggestion
CLIENT_EXE_NAME="oc"
```

I think this is better handled the same way as in the bookinfo install script.